docs/sdk/auth/fragments/android/hosted-ui.md

```diff
@@ -206,7 +246,7 @@ Note: The User Pool OIDC JWT token obtained from a successful sign-in will be fe
     </manifest>
     ```
 
-1. Attach an intent callback so that the AWSMobileClient can handle the callback and confirm sign-in or sign-out. This should be in `your.package.YourAuthIntentHandlingActivity`.
+2. Attach an intent callback so that the AWSMobileClient can handle the callback and confirm sign-in or sign-out. This should be in `your.package.YourAuthIntentHandlingActivity`.
```

Comment:
```suggestion
2. Attach an intent callback so that `AWSMobileClient` can handle the callback and confirm sign-in or sign-out. This should be in `your.package.YourAuthIntentHandlingActivity`.
```